### ISSUE TITLE
Since TLN is a Model class, not a Layer, with `output_features=1`, it's not easy to use it for classification tasks, e.g how to add one-hot encoding and `softmax` activation?

### ISSUE THREAD

**Issue by mw66** [remigenet/TLN#1]
Since TLN is a Model class, not a Layer,

class TLN(keras.Model):

https://github.com/remigenet/TLN/blob/8423fb293215b11112b2455536215e5a533bedd9/tln/tln.py#L143

with `output_features=1`, it's not easy to use it for classification tasks, e.g how to add one-hot encoding and `softmax` activation?

**Comment by remigenet** [comment]
Model class from keras inherit from Layer class, the main difference being that Model support method like fit or predict !
You can use a model as a layer in any network, for example I am using TLN here internally of another model:
https://github.com/remigenet/DeepLearningVWAP/blob/main/dvwap/deep_static_vwap.py

For your specific need, if your task is classification and you want to use softmax (so output_features would be > 1 otherwise softmax do not make sense - maybe you confuse with sigmoid if you only have one output?)

How you can do is simply:
model = Sequential([
    Input(shape=X_train.shape[1:]),
    TLN(
        output_len=n_ahead,
        output_features=n_class,
        hidden_layers=2,
        flatten_output=False,
        use_convolution=True
    ), 
    keras.layers.Activation('softmax')
])
This would return for each sample a vector (n_ahead, n_class) with softmax applied on last dimension, so having the probability of being between one of the n_class class at each future timestep

To go even further, you can even use TLN after a RNN, for example

model = Sequential([
    Input(shape=X_train.shape[1:]),
    LSTM(100, return_sequences=True),
    LSTM(100, return_sequences=True),
    TLN(
        output_len=n_ahead,
        output_features=1,
        hidden_layers=2,
        flatten_output=True,
        use_convolution=True
    )
])

Will work fine (as long as the rnn cell return the sequence of course)

**Comment by mw66** [comment]
Thanks.
